### ISSUE TITLE
Problem with recording html tags

### ISSUE THREAD

**Issue by mpstv** [rrweb-io/rrweb-snapshot#33]
Hello everyone.
(This issue duplicate https://github.com/rrweb-io/rrweb/issues/225)
I found a problem with recording html elements and styles when using angular.
The bottom line is, angular components create html markup, like this:

```html
<app-some-component _ngcontent-qah-c12="" _nghost-qah-c11="">
  <p _ngcontent-qah-c11="">some-component is visible!</p>
</app-some-component>
```

When recording events it will transform to regular div instead of an element.
You can use this [repository](https://github.com/mpstv/angular-components-rrweb-problem) to reproduce and debug.

**Comment by Yuyz0112** [comment]
Thanks for the PR!

**Comment by mpstv** [comment]
Thanks @Yuyz0112
Is there any plans to release that improvement via npm and then update rrweb?

**Comment by orelb** [comment]
Hi! thanks for the fix. We also need this, is it possible to publish a release?

**Comment by mpstv** [comment]
@Yuyz0112 we are really looking forward to the release :)

**Comment by Yuyz0112** [comment]
@orelb @mpstv sorry for the late.

This fix has been released in rrweb-snapshot@0.7.27 and rrweb@0.8.1

**Comment by mpstv** [comment]
Thank you @Yuyz0112 !
